generators/entity/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java

```diff
@@ -173,8 +173,8 @@
     private static final String <%=updatedValueName %>_CONTENT_TYPE = "image/png";
     <%_ } else if (fieldTypeBlobContent == 'text') { _%>
 
-    private static final String <%=defaultValueName %> = "<%=sampleTextString %>";
-    private static final String <%=updatedValueName %> = "<%=updatedTextString %>";
+    private static final String <%=defaultValueName %> = "";
```

Comment:
sampleTextString & updatedTextString are generated if only field type is string, so these are empty anyway.